.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+<!-- Before creating this PR, make sure to thoroughly follow the contributing guide. -->
+<!-- Add a description of the PR changes in this section. -->
+
+REPLACE ME
+
+
+## Tasks
+
+<!-- Once the PR has been created, check boxes as appropriate. -->
+
+* [ ] The [contributing guide](/.github/CONTRIBUTING.md) has been read and followed.
+* [ ] The samples added / modified have been fully tested.
```

Comment:
**Praise:** I really like this checklist item: "The samples ... have been fully tested".
It hold the contributor accountable, by default.